admin/client/TagBadge.tsx

```diff
@@ -3,10 +3,8 @@ import { observer } from "mobx-react"
 
 import { Link } from "./Link"
 
-export interface Tag {
-    id: number
-    name: string
-}
+import { Tag } from "react-tag-autocomplete"
+export { Tag } from "react-tag-autocomplete"
```

Comment:
At first glance this is confusing. Should we just import `import { Tag } from "react-tag-autocomplete"` in the file that is currently importing `Tag` from TagBadge?

Reply:
Yeah, can do that. That was mostly just a quick stopgap, since `Tag` is used by multiple files and I wanted to fix them all at once 😀

Reply:
Found an even nicer and simpler solution now ;)